### DISCUSSION TITLE
Plans to support Redis alternatives

### DISCUSSION THREAD

**Discussion by paras-malhotra** [laravel/framework#50942] [General]
Are there any plans to support Redis alternatives? This is in the context of Redis's recent move to source available, non-free licenses.

I know Memcached is supported but I guess there are better alternatives. There's of course the new fork [Valkey](https://github.com/valkey-io/valkey), [KeyDB](https://github.com/Snapchat/KeyDB) and [Garnet](https://github.com/microsoft/garnet).

**Comment by ju5t** [comment]
You can still use Redis, the license only prohibits you from selling Redis as a service. And Valkey is client-compatible so if you want to use it, you can. No changes would be needed in Laravel.
